.github/pull_request_assignment.yml

```diff
@@ -1,165 +1,207 @@
 ---
+        
 - rule:
-    # translator data-plane PR
     paths:
-      - "specification/cognitiveservices/data-plane/TranslatorText/**"
+      - "specification/@(applicationinsights|operationalinsights)/**"
     reviewers:
-      - kristapratico
-      - maririos
-
+      - akning-ms
+      - JeffreyRichter
```

Comment:
This contains bot control plane and data plane specs. It needs to separate this path into two paths, one is for control plane and the other is for data plane, then have different reviewer list for each.